session/Session.proto

```diff
@@ -64,11 +64,8 @@ message Transaction {
         oneof req {
             Open.Req open_req = 1;
             Commit.Req commit_req = 2;
-            Query.Req query_req = 3;
-            Iter.Req iterate_req = 4;
```

Comment:
Shouldn't we keep `Iter.Req`? I see that `Iter.Res` is still returned (which I expect). The `stream()` can only be sending `Transaction.Req` and receiving `Transaction.Res`, right.

Reply:
I actually intended to keep it separate from `Transaction.Req` but I've realised that this doesn't work because the stream is `Transaction.Req -> Transaction.Res`. I've put the `Req` back now.